themes/nav-community-v2/static/css/hero-section.scss

```diff
@@ -0,0 +1,100 @@
+.hero-section {
+  color: white;
+  padding: 98px 0;
+  display: grid;
+  grid-column-gap: 64px;
+  grid-auto-flow: dense;
+  grid-template-columns:1fr 1fr;
+  grid-template-areas:  "image text"
+                        "image text";
+
+  background: -moz-linear-gradient(90deg,#4AA9E4 -50%, #A34Bc1 150%);
```

Comment:
You dont need all this stuff in code. It will get auto generated by SASS

Reply:
fixed